src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -274,6 +274,31 @@ static void resultFooter(RoutingContext ctx, Integer count, String diagnostic) {
         });
   }
 
+  Future<Void> getReportPackages(Vertx vertx, RoutingContext ctx) {
+    PgCqlQuery pgCqlQuery = PgCqlQuery.query();
+    pgCqlQuery.addField(new PgCqlField("kbPackageId", PgCqlField.Type.UUID));
+    pgCqlQuery.addField(new PgCqlField("kbPackageName", PgCqlField.Type.FULLTEXT));
+    pgCqlQuery.addField(new PgCqlField("kbTitleId", PgCqlField.Type.UUID));
+
+    RequestParameters params = ctx.get(ValidationHandler.REQUEST_CONTEXT_KEY);
+    final String tenant = stringOrNull(params.headerParameter(XOkapiHeaders.TENANT));
+    final TenantPgPool pool = TenantPgPool.pool(vertx, tenant);
+
+    pgCqlQuery.parse(stringOrNull(params.queryParameter("query")));
+    String cqlWhere = pgCqlQuery.getWhereClause();
+
+    String from = packageEntriesTable(pool);
+    if (cqlWhere != null) {
+      from = from + " WHERE " + cqlWhere;
+    }
+    return streamResult(ctx, pool, null, from, "packages",
+        row -> new JsonObject()
+            .put("kbPackageId", row.getUUID(0))
+            .put("kbPackageName", row.getString(1))
+            .put("kbTitleId", row.getUUID(2))
```

Comment:
Magic number should be avoided, use the column names instead.

Reply:
There's nothing magic about that.

Reply:
How do I know that they are correct?
If an unused column is removed and then later added again the order will be different, and there is no (easy) way in PostgreSQL to reorder them when the other columns contain data.

Reply:
Using `.put("kbPackageId", row.getUUID("kbpackageid"))` always works regardless of the order of result columns.